README.md

```diff
@@ -9,6 +9,10 @@
 ## News
 - **[2025.05.25]** Our [RankLLM](https://dl.acm.org/doi/pdf/10.1145/3726302.3730331) resource paper is accepted to SIGIR 2025! 🎉🎉🎉
 
+### Important Features / Updates
++ Added support for OpenRouter API - Release [v0.25.7](docs/release-notes/release-notes-v0.25.7.md)
++ Added support for custom prompt templates with YAML files (eg: [reason rank PR](https://github.com/castorini/rank_llm/pull/306)) - Release [v0.25.0](docs/release-notes/release-notes-v0.25.0.md)
```

Comment:
Added support for custom prompt templates with YAML files - Release [v0.25.0](docs/release-notes/release-notes-v0.25.0.md). You can now integrate your own prompt and language model with just a few lines of code. Checkout the [Reasonrank integration](https://github.com/castorini/rank_llm/pull/306) as an example.